### PR TITLE
New Card Grid Desgin

### PR DESCRIPTION
**Pull request by coldmate**
- Changed old design to new card design w/ dividers 
- Added "Learn More" button that is currently linked to the experience page

### CODE REVIEW

File: pages/exp.js [hunk 1]
```diff
@@ -1,9 +1,80 @@
...
+import { useRouter } from 'next/router';
+import {Card, CardHeader, Divider, Image, CardFooter, CardBody, Button} from "@nextui-org/react";
+import COD from '../pictures/MW3_1.jpg'
+import minecraft from '../pictures/cube.jpg'  
+import sledge from '../pictures/sledgeHammer.jpg'
+import SU from '../pictures/seaRed.png'
+import snakeIcon from '../pictures/snake.jpg'
 
-// Starting to work on this page
+// *Note: There is combined tailwind css and standard css from global.css being used in this file
```
Comment (kidlatmc29) [inline pages/exp.js:11]: Good comment for clarity, thank you for adding it.

File: pages/exp.js [hunk 2]
```diff
@@ -75,6 +75,71 @@ export default function ExperiencePage() {
...
+  const handleCardPress = (url) => {
+    router.push(url)
+  };
+
+  const list = [
+    {
+      url: "http://localhost:3000/exp",
+      company: "Activision - Sledgehammer Games",
+      position: "Associate Gameplay Engineer",
+      img: COD, // can't directly use the path, img won't show up on live
```
Comment (kidlatmc29) [inline pages/exp.js]: Do you know the reason why a direct path won't work for imgs?

**Review by kidlatmc29 — APPROVED** [review]

File: pages/exp.js (hunk 2 above)
Comment (coldmate) [reply pages/exp.js]: Found the issue, the images are stored in a separate file, pictures, when stored in the public directory (which is built when creating a nextjs project), nextjs can properly load images in

Comment (coldmate) [reply pages/exp.js]: Changing it now

**Pull request merged**